src/hooks/useMapKeyboardControls.ts

```diff
@@ -1,18 +1,13 @@
 import { useCallback } from "react";
 
+import { KeyboardShortcutKeys } from "settings";
+
 import { ReactHook } from "types/utility";
 
 import { useMousetrap } from "hooks/useMousetrap";
 import { useUser } from "hooks/useUser";
 
-export const MovementKeys = {
-  up: ["up", "w"],
-  down: ["down", "s"],
-  left: ["left", "a"],
-  right: ["right", "d"],
-};
-
-interface UseKeyboardControlsProps {
+interface UseMapKeyboardControlsProps {
```

Comment:
Seems this code predates our current style/pattern of exporting all of the props types:

```suggestion
export interface UseMapKeyboardControlsProps {
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1416/commits/2dc066902d7fe56f50eb54583cc33294a7f246db